src/devices/bus/isa/ibm_speech.h

```diff
@@ -0,0 +1,88 @@
+// license:BSD-3-Clause
+// copyright-holders:Katherine Rohl
+
+#ifndef MAME_BUS_ISA_IBM_SPEECH_H
+#define MAME_BUS_ISA_IBM_SPEECH_H
+
+#pragma once
+
+#include "isa.h"
+#include "machine/i8255.h"
+#include "machine/pit8253.h"
+#include "sound/tms5220.h"
+#include "sound/hc55516.h"
+#include "sound/spkrdev.h"
+#include "speaker.h"
+
+class isa8_ibm_speech_device :
+		public device_t,
+		public device_isa8_card_interface
+{
+public:
+	// construction/destruction
+	isa8_ibm_speech_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+
+	uint8_t audio_control_latch_r();
+	void audio_control_latch_w(uint8_t data);
+
+	uint8_t shift_register_r();
+	void shift_register_w(uint8_t data);
+
+protected:
+	// device-level overrides
+	virtual void device_start() override;
+	virtual void device_reset() override;
+	virtual void device_reset_after_children() override;
+	virtual void device_add_mconfig(machine_config &config) override;
+
+	// optional information overrides
+	virtual const tiny_rom_entry *device_rom_region() const override;
```

Comment:
Please stop copy-pasting these confusing comments.  Just group these together and comment them “device_t implementation” or something.